stdlib/src/python/object.mojo

```diff
@@ -1102,6 +1102,28 @@ struct PythonObject(
         """
         return self._call_zero_arg_method("__invert__")
 
+    fn __contains__(self, rhs: PythonObject) raises -> Bool:
```

Comment:
To clarify, in one of your comments, you're asking about the return type here, correct?  E.g. `Bool` vs `PythonObject`?  I'm fine with `Bool` as-is.  What do you think?

Reply:
Yes, the return type of the function :+1: 

It should be good, `Bool` can probably implicitly convert to `PythonObject` if user do chaining.

(but it is something to keep in mind for the future)

Reply:
Exactly, I had the composability question in mind, and I think we're fine with `Bool` for now.